src/main/java/jp/ac/keio/bio/fun/xitosbml/cli/CLI_Filetype.java

```diff
@@ -0,0 +1,43 @@
+package jp.ac.keio.bio.fun.xitosbml.cli;
+
+import java.io.File;
+import java.util.concurrent.Callable;
+
+import picocli.CommandLine;
+import picocli.CommandLine.Command;
+import picocli.CommandLine.Option;
+
+@Command(name = "Running XitoSBML-CLI", mixinStandardHelpOptions = true, description = "Saves VIA File option", version = "1.0")
+
+public class CLI_Filetype implements Callable<Integer> {
```

Comment:
Why did we create a new class here? Why do we need to separate with cliRun?

Reply:
I guess we could do this with the same class (CliRun.java). I would have to see though, how that would work out.

Reply:
I have tried to implement this in https://github.com/Medha-B/XitoSBML/pull/10/commits/e4c9f6c643b5987abd7d9e8fa72470a2762e78e7

Reply:
If we have the main method in `CliRun.java`, what is this class used for?

Reply:
CLI_Filetype.java has no separate use at the moment (can be disposed of) since we can now process the dataset using CliRun.java. Is this not what you wanted (same class for the dataset as well as single image input)?